src/gpl/src/nesterovPlace.cpp

```diff
@@ -374,6 +383,21 @@ NesterovPlace::updateGradients(
   debugPrint(log_, GPL, "replace", 3, "updateGrad:  WireLengthGradSum: {:g}", wireLengthGradSum_);
   debugPrint(log_, GPL, "replace", 3, "updateGrad:  DensityGradSum: {:g}", densityGradSum_);
   debugPrint(log_, GPL, "replace", 3, "updateGrad:  GradSum: {:g}", gradSum);
+
+  // sometimes wirelength gradient is zero when design is too small
+  if( wireLengthGradSum_ == 0 ) {
+    wireLengthCoefX_ *= 0.5;
+    wireLengthCoefY_ *= 0.5;
+    baseWireLengthCoef_ *= 0.5;
+    log_->warn(GPL, 320, "sum(WL gradient) = 0, try again with {:g} {:g}", 
```

Comment:
1) Please make the warning a little easier to understand for laymen and avoid acronyms.

2) Also if this is something that a user can't do anything about, I suggest making this an `INFO`. If there are parameters or arguments that a user can change, then it can stay a `WARN` but there should be details on what the user should change.

3) Try -> Trying

Reply:
This should just be a debug print since no user will ever care about this.

Reply:
ok. I changed it as debugPrint